### PR TITLE
Change string positions to listings / Clarify "Archives" Language in Expired Listings / Filled Positions Settings

### LINKED ISSUE #2491: Confusion Between "Listings" and "Positions" in Settings
### Is your feature request related to a problem? Please describe

The language of "Positions" and "Listings" is potentially confusing on the WP Job Manager settings:

Annotation-Annotation on 2023-07-05 at 11-25-24 png

We use "Listings" throughout but have one place where we refer to "Positions".

### Describe the solution you'd like

It might be clearer if we changed the to make it more clear that "Filled Positions" are "Filled Listings".

Perhaps using the text: `Job listings marked as "filled" will not display in your archives.`

### Describe alternatives you've considered

Eliminate the "positions" reference entirely and replace with "listings"

### Additional context

This was raised on 6482026-zen

### LINKED ISSUE #2492: Clarify "Archives" Language in Expired Listings / Filled Positions Settings
### Is your feature request related to a problem? Please describe

Screenshot on 2023-07-05 at 11-52-52(1)

The language here is a bit unclear. "Archives" could potentially mean the backend of the site, or the frontend. What we are referring to here is the frontend. The settings don't change what displays in WP Admin.

### Describe the solution you'd like

It would be nice to clarify that:

- These settings affect the frontend only
- Jobs will always be visible via the backend
- Perhaps instead of "archives" we could use "job listings"?
- We could clarify that when "filled jobs" are hidden, their individual page remains visible. 
- The use of "search" is confusing. Both options, hide the the jobs from search. 

### Additional context

- Raised on 6482026-zen

### PR DESCRIPTION
**Pull request by masteradhoc**
Fixes #2491, Fixes #2492

### Changes Proposed in this Pull Request

* Change appearance of string positions to listings

### Testing Instructions

* Go through all screens and check the appearance of positions and check if they are changed to listings after the PR

### Release Notes

* Change positions to listings for consistency

### CODE REVIEW

File: includes/admin/class-wp-job-manager-settings.php [hunk 1]
```diff
@@ -159,18 +159,18 @@ protected function init_settings() {
 						[
 							'name'       => 'job_manager_hide_filled_positions',
 							'std'        => '0',
-							'label'      => __( 'Filled Positions', 'wp-job-manager' ),
-							'cb_label'   => __( 'Hide filled positions', 'wp-job-manager' ),
-							'desc'       => __( 'Filled positions will not display in your archives.', 'wp-job-manager' ),
+							'label'      => __( 'Filled Listings', 'wp-job-manager' ),
+							'cb_label'   => __( 'Hide filled listings', 'wp-job-manager' ),
+							'desc'       => __( 'Filled job listings will not be shown to the users on the jobboard. The individual page remains visible.', 'wp-job-manager' ),
```
Comment (gikaragia) [inline includes/admin/class-wp-job-manager-settings.php]:
```suggestion
							'desc'       => __( 'Filled job listings will not be included in search results, sitemap and archives.', 'wp-job-manager' ),
```

File: includes/admin/class-wp-job-manager-settings.php [hunk 2]
```diff
@@ -159,18 +159,18 @@ protected function init_settings() {
...
 						],
 						[
 							'name'       => 'job_manager_hide_expired',
 							'std'        => get_option( 'job_manager_hide_expired_content' ) ? '1' : '0', // back compat.
-							'label'      => __( 'Hide Expired Listings', 'wp-job-manager' ),
-							'cb_label'   => __( 'Hide expired listings in job archives/search', 'wp-job-manager' ),
-							'desc'       => __( 'Expired job listings will not be searchable.', 'wp-job-manager' ),
+							'label'      => __( 'Expired Listings', 'wp-job-manager' ),
+							'cb_label'   => __( 'Hide expired listings', 'wp-job-manager' ),
+							'desc'       => __( 'Expired job listings will not be shown to the users on the jobboard. The individual page remains visible.', 'wp-job-manager' ),
```
Comment (gikaragia) [inline includes/admin/class-wp-job-manager-settings.php]:
```suggestion
							'desc'       => __( 'Expired job listings will not be shown to the users on the job board., 'wp-job-manager' ),
```

File: templates/job-filters.php [hunk 3]
```diff
@@ -39,7 +39,7 @@
 		<?php if( apply_filters( 'job_manager_job_filters_show_remote_position', get_option('job_manager_enable_remote_position', true ), $atts ) ) : ?>
 			<div class="search_remote_position">
 				<input type="checkbox" class="input-checkbox" name="remote_position" id="remote_position" placeholder="<?php esc_attr_e( 'Location', 'wp-job-manager' ); ?>" value="1" <?php checked(! empty( $remote_position ) ); ?> />
-				<label for="remote_position" id="remote_position_label"><?php esc_html_e( 'Remote positions only', 'wp-job-manager' ); ?></label>
+				<label for="remote_position" id="remote_position_label"><?php esc_html_e( 'Remote listings only', 'wp-job-manager' ); ?></label>
```
Comment (gikaragia) [inline templates/job-filters.php]: Although I can see a benefit in being consistent, I would prefer not to change a frontend facing label for the sake of it. I also think that positions is a better word here.

```suggestion
				<label for="remote_position" id="remote_position_label"><?php esc_html_e( 'Remote positions only', 'wp-job-manager' ); ?></label>
```

**Comment by gikaragia** [comment]
Thank you for the contribution @masteradhoc, looks good overall! I needed to do some changes and I raised another PR [here](https://github.com/Automattic/WP-Job-Manager/pull/2639).

File: languages/wp-job-manager.pot [hunk 4]
```diff
@@ -572,15 +572,15 @@ msgid "Page numbered links"
 msgstr ""
```
Comment (gikaragia) [inline languages/wp-job-manager.pot:572]: We don't update the pot file during development. It is automatically generated during release.

**Comment by masteradhoc** [comment]
Thanks @gkaragia!

File: languages/wp-job-manager.pot (hunk 4 above)
Comment (masteradhoc) [reply languages/wp-job-manager.pot:572]: Thanks for the input!

**Pull request merged**